frontend/packages/operator-lifecycle-manager/src/components/clusterserviceversion.tsx

```diff
@@ -1229,6 +1229,7 @@ export const ClusterServiceVersionDetailsPage: React.FC<ClusterServiceVersionsDe
     group: SubscriptionModel.apiGroup,
     resource: SubscriptionModel.plural,
     verb: 'list',
+    namespace: ns,
```

Comment:
Is the subscription always in the same namespace? I think this will show the tab sometimes when the user doesn't have permission for global operators.

Reply:
@spadgett the subscription is create in the namespace into which the operator is installed.

Reply:
@jhadvig Is that true for copied CSVs? The CSV is not necessarily in the same namespace.

Reply:
>Is that true for copied CSVs? The CSV is not necessarily in the same namespace.

@spadgett based on the convo with @awgreene and @dmesser the subscription is create in the namespace into which the operator is installed, even if CSV are copied all around the cluster, end copied CSVs are enabled

Reply:
@spadgett Is correct. The Subscription will be in the namespace defined by an annotation on the CSV, not necessarily the current active namespace.